src/server/_common.py

```diff
@@ -18,12 +20,37 @@ def _get_db() -> Connection:
         g.db = conn
     return g.db
 
+def log_and_raise_exception(message):
```

Comment:
method name should include something about the only exception it throws
```suggestion
def log_and_raise_validation_exception(message):
```